sourcegit/sync.py

```diff
@@ -28,6 +31,21 @@ def __init__(self, pagure_user_token, pagure_package_token, pagure_fork_token) -
         #        on the class
         self._tempdirs = []
 
+    def reset_checks(self, full_name: str, pr_id: str, checks_list: list):
```

Comment:
Are we going to use type-annotations? I am definitely not against it.

Reply:
I just wanted to try it, because I saw one in one of the methods (done by you?); I think it could be really beneficial.

Reply:
The pycharm sometimes add the annotations. I've never tried that as well.